test/Sign.Cli.Test/Options/UriOptionTests.cs

```diff
@@ -20,5 +20,32 @@ public void Option_WhenValueFailsToParse_HasError()
         {
             VerifyHasError("3");
         }
+
+        [Theory]
+        [InlineData("http://domain.test")]
+        [InlineData("https://domain.test")]
+        public void Option_WithShortOptionAndValidUrl_ParsesValue(string url)
+        {
+            Uri expectedValue = new(url, UriKind.Absolute);
+
+            Verify($"{ShortOption} {expectedValue.OriginalString}", expectedValue);
+        }
+
+        [Theory]
+        [InlineData("//domain.test")]
+        [InlineData("/path")]
+        [InlineData("file:///file.bin")]
+        public void Option_WithShortOptionAndInvalidUrl_HasErrors(string invalidUrl)
+        {
+            VerifyHasError($"{ShortOption} {invalidUrl}");
```

Comment:
Should we test if outputting expected error message something like 

`Assert.Equal("Unsupported value for --url. The value must be an absolute HTTP or HTTPS URL."?, result.Errors.First()); `

Reply:
@erdembayar, I created a [separate issue](https://github.com/dotnet/sign/issues/544) to track that feedback.  I don't think it should hold up this PR, as I believe that the change, if applied, should be applied to all tests, not just new tests.